python/ray/serve/_private/logging_utils.py

```diff
@@ -47,6 +47,13 @@ def access_log_msg(*, method: str, status: str, latency_ms: float):
     return f"{method.upper()} {status.upper()} {latency_ms:.1f}ms"
 
 
+def get_logger_file_path() -> Optional[str]:
```

Comment:
```suggestion
def get_component_logger_file_path() -> Optional[str]:
```
nit: match name below